README.md

```diff
@@ -0,0 +1,36 @@
+# Install OMERO on CentOS 7 or Ubuntu 18.04 using Ansible
+
+1. Install Ansible (optionally in a virtualenv):
```

Comment:
We have now opted to recommend to use conda in many repositories. I think we should follow the same path for this repo.

Reply:
I've mentioned a virtualenv and conda. Since this is aimed at sysadmins and not developers I don't think we should focus so much on conda.